python/tskit/trees.py

```diff
@@ -3014,6 +3014,56 @@ def __getstate__(self):
     def __setstate__(self, tc):
         self._ll_tree_sequence = tc.tree_sequence().ll_tree_sequence
 
+    def __eq__(self, other):
+        return self.tables == other.tables
+
+    def equals(
+        self,
+        other,
+        *,
+        ignore_metadata=False,
+        ignore_ts_metadata=False,
+        ignore_provenance=False,
+        ignore_timestamps=False,
+    ):
+        """
+        Returns True if  `self` and `other` are equal. By default, two tree sequences
+        are considered equal if their underlying tables'
+
```

Comment:
We could reduce a bit of duplication and say that ``equals`` is defined by the ``equals`` of the underlying table collections and link to that documentation.

Reply:
Done.